assets/sass/v2/_device-challenge.scss

```diff
@@ -0,0 +1,27 @@
+.siw-main-view {
+  .device-challenge-poll {
+    .o-form-content {
+      .okta-form-subtitle {
+        font-weight: 600;
+      }
+    }
+    .o-form-fieldset-container {
```

Comment:
this is just not ideal.
either get ride of it if we don't have to or override on specific class for the view if possible.